content/help/faq/security/k8s-health-checks.md

```diff
@@ -3,10 +3,10 @@ title: How can I use Kubernetes liveness and readiness for service health check
 weight: 50
 ---
 If mutual TLS is enabled, http and tcp health checks from the kubelet will not
-work since they do not have Istio-issued certs. A workaround is to
+work since kubelet does not have Istio-issued certs. A workaround is to
```

Comment:
since kubelet -> since the kubelet
certs -> certificates

Reply:
done